source/_integrations/garmin_connect.markdown

```diff
@@ -0,0 +1,126 @@
+---
+title: Garmin Connect
+description: Instructions on how to configure the Garmin Connect integration for Home Assistant.
+logo: garmin_connect.png
+ha_category:
+  - Health
+ha_iot_class: Cloud Polling
+ha_release: 0.19
+ha_codeowners:
+  - '@cyberjunky'
+---
+
+The Garmin Connect sensor allows you to expose data from [Garmin Connect](https://connect.garmin.com) to Home Assistant.
+
+Enable the sensor by adding the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: garmin_connect
+    email: GARMIN_CONNECT_EMAIL
+    password: GARMIN_CONNECT_PASSWORD
+    resources:
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

Reply:
Understood, and fixed.